packages/attest/src/endpoints.ts

```diff
@@ -45,6 +37,20 @@ export const signingEndpoints = (sigstore?: SigstoreInstance): Endpoints => {
     case PUBLIC_GOOD_ID:
       return SIGSTORE_PUBLIC_GOOD
     case GITHUB_ID:
-      return SIGSTORE_GITHUB
+      return buildGitHubEndpoints()
+  }
+}
+
+function buildGitHubEndpoints(): Endpoints {
+  const serverURL = process.env.GITHUB_SERVER_URL ?? `https://github.com`
+  let url = serverURL.replace('https://', '')
+
+  if (url === 'github.com') {
+    url = 'githubapp.com'
+  }
+  const endpoints: Endpoints = {
+    fulcioURL: `https://fulcio.${url}`,
+    tsaServerURL: `https://timestamp.${url}`
   }
+  return endpoints
```

Comment:
Should be able to return a literal here:
```suggestion
  return {
    fulcioURL: `https://fulcio.${url}`,
    tsaServerURL: `https://timestamp.${url}`
  }
```